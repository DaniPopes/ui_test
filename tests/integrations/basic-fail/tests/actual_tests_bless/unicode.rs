#![deny(confusable_idents)]

// Test that a missing annotation causing a
// ui_test diagnostic will not ICE due to byte vs char offsets.

fn main() {
    let Ě电脑 = 1;
}
