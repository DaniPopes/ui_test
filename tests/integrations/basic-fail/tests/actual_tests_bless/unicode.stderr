warning: unused variable: `Ě电脑`
 --> tests/actual_tests_bless/unicode.rs:7:9
  |
7 |     let Ě电脑 = 1;
  |         ^^^^^ help: if this is intentional, prefix it with an underscore: `_Ě电脑`
  |
  = note: `#[warn(unused_variables)]` on by default

warning: variable `Ě电脑` should have a snake case name
 --> tests/actual_tests_bless/unicode.rs:7:9
  |
7 |     let Ě电脑 = 1;
  |         ^^^^^ help: convert the identifier to snake case: `ě电脑`
  |
  = note: `#[warn(non_snake_case)]` on by default

warning: 2 warnings emitted

